pkg/client/events/replay_client.go

```diff
@@ -181,8 +211,6 @@ func (rClient *replayClient[T, R]) retryPublishEventsFactory(ctx context.Context
 		}
 
 		// NB: must cast back to generic observable type to use with Map.
-		// client.BlocksObservable cannot be an alias due to gomock's lack of
-		// support for generic types.
 		eventsBz := observable.Observable[either.Either[[]byte]](eventsBzObs)
```

Comment:
Wdyt about renaming this to `eventsBzObs`, as it's used more than the variable that currently uses that name, and renaming that one to something like `eventsBytesObs`?